functions.py

```diff
@@ -482,3 +482,106 @@ def add_data_from(pythonClient,data_csv):
         if "DUPLICATE" in str(e):
             logging.error("Duplicate data")
         exit()
+
+
+def getVariablesbyURI(uri="", pythonClient=""):
```

Comment:
`def getVariablesbyURI(uri=None, pythonClient=None):`

Reply:
It's better to use None instead an empty character variable to define these default values

Reply:
Corrected in [commit](https://github.com/OpenSILEX/opensilexClientPython/pull/1/commits/d4265714dde879b5f17d55e913a55ab232b393d6)